imgui.go

```diff
@@ -1227,6 +1279,56 @@ func ColumnsCount() int {
 	return int(C.iggGetColumnsCount())
 }
 
+// BeginDragDropSource opens the scope for current draw and drop source.
+// Call when current ID is active.
+// When this returns true you need to: a) call SetDragDropPayload() exactly once, b) you may render the payload visual/description, c) call EndDragDropSource()
+func BeginDragDropSource(flags int) bool {
+	return C.iggBeginDragDropSource(C.int(flags)) != 0
+}
+
+// SetDragDropPayload sets the payload for current draw and drop source.
+// type is a user defined string of maximum 32 characters.
+// Strings starting with '_' are reserved for dear imgui internal types.
+// Data is copied and held by imgui.
+func SetDragDropPayload(type_ string, payload string, cond Condition) bool {
```

Comment:
In the library, the payload (called `data` in the API by the way) is handled rather as an arbitrary byte array. It should be exposed as such here as well.